src/main/java/bookstore/dto/CreateBookRequestDto.java

```diff
@@ -1,14 +1,22 @@
 package bookstore.dto;
 
+import jakarta.validation.constraints.NotNull;
+import jakarta.validation.constraints.PositiveOrZero;
+import jakarta.validation.constraints.Size;
 import java.math.BigDecimal;
 import lombok.Data;
 
 @Data
 public class CreateBookRequestDto {
```

Comment:
let's convert this class to record

Reply:
якщо памятаєте я писав що не знаю як - на теоріїї по відео не бачив

Reply:
[Link to example](https://www.baeldung.com/java-record-keyword)

```suggestion
public record CreateBookRequestDto(@NotNull String title,
                                   @NotNull String author,
                                   @NotNull @Size(min = 5, max = 10)
                                   String isbn,
                                   @NotNull @PositiveOrZero BigDecimal price,
                                   String description,
                                   String coverImage) {

}
```